api/apps/services/structure_graph_common.py

```
@@ -317,14 +386,16 @@ async def keyword_subgraph(index_name, kb_id, embd_mdl, base_entity_condition, k
         embedding_data=vec,
         embedding_data_type="float",
         distance_type="cosine",
-        topn=1,
+        topn=GRAPH_TOP_ENTITIES,
         extra_options={"similarity": 0.0},
     )
     top_fields = GRAPH_ENTITY_FIELDS + ["compilation_template_ids", "compile_kwd", "compilation_template_kind_kwd"]
```

Ruff (0.16.1)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)